### PR TITLE
Old map features + updates from box!! 

### PR DESCRIPTION
**Pull request by yoyobatty**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: YoYoBatty
add: Added back in old hippie maint areas in the southwest solar maint. This includes ghetto botany.
add: Added back in old genetics/cloning.
add: A proper security checkpoint to escape.
add: Tons of cosmetic additions. 
add: Replaced department clothing wardrobes with their proper vending machines.
add: Added some "grille or trash maint spawners" in certain areas.
add: More catwalks go around the southwest of station to engineering.
add: Adds some cycle linked dir controllers to airlocks in AI sat.
add: Moves vending machines in the chapel to library to make space for escape sec checkpoint.
add: Air vent/scrubbers to some maint areas.
add: Kitchen now has a bar access door.
del: Removed some ugly/old cosmetics.
fix: Anchored down the wired airlock assemblies by viro/atmos and russian room maints.
fix: Area strings are now to assigned to apcs that didn't have them.
tweak: Vault door icon is now properly assigned in the map editor + uses airlock lock helper
tweak: Adjusted the area for arrivals sec checkpoint to encompass the light tubes around the structure in order to take lighting load off of the arrivals apc.
tweak: Replaced the last couple window + grille combos with window spawners.
tweak: Changed certain airlocks access requirements in science maint to let scientist rummage about. 
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

here are some screenshots

image

moves some in wall pipes in gravity gen to go beneath floor because standards!
image

image

image

image

image

image

Little note, some of these screenshots are slightly out dated and have had some work done to them.

### CODE REVIEW

**Review by Lxtoofonea — APPROVED** [review]
Y E S
but i would also remove the maint airlock above arrivals, don't want no assistance to avoid my sweet sweet paperwork (^:

**Comment by Carbonhell** [comment]
Holy shit nice job, +1

**Comment by Carbonhell** [comment]
This is the perfect pr to be #8000 kek

**Comment by earthmoonjune** [comment]
+1

**Comment by Jujumatic** [comment]
dude is this what old genetics was like? that's AMAZING

**Comment by DaydreamPuppeh** [comment]
Yes please +1

File: hippiestation/code/game/objects/structures/crates_lockers/closets.dm [hunk 1]
```diff
@@ -6,30 +6,4 @@ Sorry for doing this, but apparently the Hippie community hates art and new thin
...
-				add_overlay("off")
-
-	else
-		layer = BELOW_OBJ_LAYER
-		if(icon_door_override)
-			add_overlay("[icon_door]_open")
-		else
-			add_overlay("[icon_state]_open")
\ No newline at end of file
+	icon_hippie = 'hippiestation/icons/obj/closet.dmi'
```
Comment (Carbonhell) [inline hippiestation/code/game/objects/structures/crates_lockers/closets.dm:9]: Does this also fix some stuff like coffins being invisible?

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**